sdk/tools/cabman/CCFDATAStorage.cxx

```diff
@@ -51,12 +51,17 @@ CCFDATAStorage::~CCFDATAStorage()
 * Creates the file
 *
 * @return
-* Status of operation
+*     Status of operation
 */
 ULONG CCFDATAStorage::Create()
 {
     if ((FileHandle = tmpfile()) == NULL)
-        return CAB_STATUS_CANNOT_CREATE;
+    {
+        /* Workaround for breakage on some Windows system */
+        FileHandle = fopen(tmpnam(NULL) + 1, "wb");
+        if (FileHandle == NULL)
+            return CAB_STATUS_CANNOT_CREATE;
+    }
```

Comment:
Do we really know what causes this breakage?

Reply:
first `tmpfile()` fails.

Reply:
For some reason tmpfile() fails. Only happens with GCC builds, only on some Windows 10 systems. This fixes it for me. Everything else is left as an exersize to the reader ;-)